source/Server/Configuration/DirectoryServicesConfigurationResource.cs

```diff
@@ -9,8 +9,8 @@ class DirectoryServicesConfigurationResource : ExtensionConfigurationResource
     {
 
         public const string ActiveDirectoryContainerDescription = "Set the active directory container used for authentication.";
-
-        public const string AuthenticationSchemeDescription = "When Domain authentication is used, specifies the scheme (Basic, Digest, IntegratedWindowsAuthentication, Negotiate, Ntlm). You will need to restart all Octopus Server nodes in your cluster for these changes to take effect.";
+        
+        public const string AuthenticationSchemeDescription = "When Domain authentication is used, specifies the scheme (Basic, Digest, IntegratedWindowsAuthentication, Negotiate, Ntlm). You will need to restart all Octopus Server nodes in your cluster for these changes to take effect. Please note that using Negotiate or IntegratedWindowsAuthentication [may require additional server configuration](https://g.octopushq.com/AuthAD) in order to work correctly.";
```

Comment:
Do links get rendered on the ui as clickable links?

Reply:
:man_facepalming: forgot I left that there - thanks for spotting it.